src/components/Nav/BeforeSearch.js

```diff
@@ -0,0 +1,275 @@
+                      }월 ${startDate.getDate()}일 ~ ${
+                        endDate.getMonth() + 1
+                      }월 ${endDate.getDate()}일`
+                    : '언제든 일주일'}
+                </Menu>
+              </SmallSearchMenuContainer>
+              <Separation />
+              <SmallSearchMenuContainer>
+                <OnlineMenu>
+                  {guest !== 0 ? `성인 ${guest}명` : '게스트 추가'}
+                </OnlineMenu>
+                <SearchCircle>
+                  <i class="bx bx-search" />
+                </SearchCircle>
+              </SmallSearchMenuContainer>
+            </SmallSearchBox>
+          </MenuContainer>
+          <UserSection>
+            <UserTextContainer>호스트되기</UserTextContainer>
+            <UserIconContainer>
+              <i class="bx bx-world" />
+            </UserIconContainer>
+            <UserInfoContainer>
+              <UserInfoMenu>
+                <i class="bx bx-menu" />
+              </UserInfoMenu>
+              <UseerInfoIcon>
+                <i class="bx bxs-user-circle" />
+              </UseerInfoIcon>
+            </UserInfoContainer>
+          </UserSection>
+        </TopNavSection>
+      </OnClickSearchSection>
+    </SectionBefore>
+  );
+};
+
+export default BeforeSearch;
+
+const fadeIn = keyframes`
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+
+`;
+
+const shadow = keyframes`
+  from {
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+  }
+  to {
+    box-shadow: rgba(0, 0, 0, 0.2) 0px 4px 12px;
+  }
+`;
+
+const SectionBefore = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  opacity: 0;
+  z-index: 102;
+  background-color: white;
+
+  &.toggle_open {
+    opacity: 1;
+    animation-duration: 0.25s;
+    animation-timing-function: ease-out;
+    animation-name: ${fadeIn};
+    animation-fill-mode: both;
+  }
+`;
+
+const OnClickSearchSection = styled.div`
+  width: 100%;
+  max-width: 1760px;
+  margin: 0 auto;
+  padding: 0 80px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+`;
+
+const TopNavSection = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+  padding-bottom: 10px;
+`;
+
+const LogoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  width: 437px;
+`;
+
+const Logo = styled.img`
+  background-color: white;
+  width: 102px;
+  height: 50px;
+  object-fit: contain;
+`;
+
+const MenuContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  width: 504px;
+`;
+
+const SmallSearchBox = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
+  width: fit-content;
+  height: auto;
+  padding: 0 10px;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  border-top-left-radius: 24px;
+  border-bottom-left-radius: 24px;
+  border-top-right-radius: 24px;
+  border-bottom-right-radius: 24px;
```

Comment:
```suggestion
  border-radius: 24px;
```
네 방향이 모두 같은 픽셀 만큼 둥그래져야 한다면
이렇게 하나로 줄 수 있습니다~

Reply:
오호..